contract/src/test/java/io/neow3j/contract/PolicyContractTest.java

```diff
@@ -0,0 +1,172 @@
+package io.neow3j.contract;
+
+import static io.neow3j.contract.ContractTestHelper.setUpWireMockForCall;
+import static io.neow3j.contract.ContractTestHelper.setUpWireMockForInvokeFunction;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import io.neow3j.crypto.ECKeyPair;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.http.HttpService;
+import io.neow3j.transaction.Cosigner;
+import io.neow3j.transaction.Transaction;
+import io.neow3j.utils.Numeric;
+import io.neow3j.wallet.Account;
+import io.neow3j.wallet.Wallet;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class PolicyContractTest {
+
+    @Rule
+    public WireMockRule wireMockRule = new WireMockRule();
+
+    private Neow3j neow3j;
+    private Wallet consensusWallet;
+
+    private static final ScriptHash CONSENSUS_MULTISIG_SCRIPT_HASH =
+            new ScriptHash("55b842d631f43f23257a27992ac2b53169a4fe00");
+    private static final String CONSENSUS_MULTISIG_ADDRESS = "AFs8hMHrS8emaPP4oyTuf5uKPuAW6HZ2DF";
+    private static final byte[] VERIFICATION_SCRIPT =
+            Numeric.hexStringToByteArray("110c2102c0b60c995bc092e866f15a37c176bb59b7ebacf069ba94c0ebf561cb8f956238110b41c330181e");
+
+    @Before
+    public void setUp() {
+        // Configuring WireMock to use default host and port "localhost:8080".
+        WireMock.configure();
```

Comment:
Just a side question (maybe @csmuller can also help here): can we configure `WireMock` to use a random port each time (for each test suite)? Then, we could fetch the random port and inject it to the `HttpService` for each time, instead of using a static/fixed one (8080). This way we would prevent sporadic failures to tests... sometimes I see that the WireMock port is already in use, somehow...

Reply:
I configured `WireMock`, such that it uses a dynamic port and then inject it to the `HttpService`. Have a look at it and after reviewing it, we could add that configuration also to other test classes that use `WireMock`.